### DISCUSSION TITLE
我觉得要是能用这个出一个  http://www.vueadminwork.com/index 或  https://admin.spicyboy.cn/#/home/index 这样漂亮的 后台管理模板项目就好了

### DISCUSSION THREAD

**Discussion by mei-rune** [tolking/element-pro-components#561] [General]
我觉得要是能用这个出一个  http://www.vueadminwork.com/index 或  https://admin.spicyboy.cn/#/home/index 这样漂亮的 后台管理模板项目就好了,

这也好证明这个项目的实用性

**Comment by tolking** [comment]
有一个简单的 [Template](https://github.com/tolking/element-admin-template)

我也想创建一个漂亮的后台，可是自己设计不行，也找不到能够参与的开源设计师

**Reply by mei-rune** [reply]
我想一个漂亮的，  我是后端程序员，做前端更不行。
